src/web/installer.go

```diff
@@ -178,37 +177,35 @@ func installerForExe(c echo.Context) error {
 }
 
 func analytics(cid string, version InstallerVersion, c echo.Context) {
-	data := url.Values{}
-	data.Set("v", "1")
-	data.Set("t", "event")
-	data.Set("tid", "UA-143397381-1")
-	data.Set("cid", cid)
-	data.Set("ds", "backend")
-	data.Set("ec", "installer")
-	data.Set("ea", "download")
-	data.Set("el", version.getEXT())
-	data.Set("ua", c.Request().UserAgent())
-
 	forward := strings.Split(c.Request().Header.Get("X-FORWARDED-FOR"), ",")[0]
-	if forward != "" {
```

Comment:
can we preserve this logic? when we can't get their forwarded IP we should omit the `uip` param, not pass it as empty. 

as they say, https://discordapp.com/channels/208753003996512258/308653317834145802/661347628818235422

Reply:
Sure